src/Mailer/Message/CitizenProjectApprovedSummaryMessage.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace AppBundle\Mailer\Message;
+
+use AppBundle\Entity\Adherent;
+use Ramsey\Uuid\Uuid;
+
+final class CitizenProjectApprovedSummaryMessage extends Message
+{
+    public static function create(Adherent $adherent, string $template): self
+    {
+        $message = new self(
+            Uuid::uuid4(),
+            '289976',
+            $adherent->getEmailAddress(),
+            $adherent->getFullName(),
+            ' [Projets citoyens] Les projets citoyens près de chez vous !',
+            [
+                'target_firstname' => self::escape($adherent->getFirstName()),
+                'citizen_project_list' => self::escape($template),
+            ]
+        );
+
+        $message->setSenderEmail('projetscitoyens@en-marche.fr');
```

Comment:
Another reason to use a message factory, this should be injected.

Reply:
i agree.
this comment should be for #1600